CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.16.0 - 2016-11-17
+
+* Added ability to run suite or test in specified browsers and skip in other silently through new method `only`
```

Comment:
Add ability to run specific tests only in specific browser without marking test as skipped in other browsers. See method `[only](link/to/doc)`